src/components/biggive-footer/biggive-footer.tsx

```diff
@@ -68,5 +76,155 @@ export class BiggiveFooter {
         </div>
       </footer>
     );
+    }
+
```

Comment:
Just like the present main menu, the content bellow is copied from https://github.com/thebiggive/donate-frontend/blob/004a6d9b6f631726a0597536eabd400195fb5b7d/src/app/app.component.html#L1 and then tweaked slightly to work outside Angular.